app/content/profile/index.js

```diff
@@ -82,9 +82,8 @@ function Index(props) {
     ["drillInfo"],
   ];
 
-  const initialSnapPoints = useMemo(() => [355, 455, 730], []);
-  const expandedSnapPoints = useMemo(() => [460, 570, 860], []); // Adjusted snap points for expanded content
-  const [snapPoints, setSnapPoints] = useState(initialSnapPoints);
+  const [snapPoints, setSnapPoints] = useState(["60%"]);
+  const [isTyping, setIsTyping] = useState(false);
```

Comment:
O ye this isTyping state, what's it used for? It's set but not used

Reply:
I think it's something Artem set up in his PR

Reply:
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/82061589/7ac04036-8e20-48d3-a6b6-fdec5e0f2ab6)

It looked to all be working the same after removing `isTyping` flag so I think it's all good, seems like `keyboardDidShowListener` and `keyboardDidHideListener` is what's used. I did message Artem on Discord just in case it is important.